components/x-article-save-button/stories/index.js

```diff
@@ -0,0 +1,11 @@
+const { ArticleSaveButton } = require('../');
+
+exports.component = ArticleSaveButton;
+exports.package = require('../package.json');
+exports.knobs = require('./knobs');
+exports.dependencies = {
+	'o-fonts': '^3.0.0'
```

Comment:
If you switch this to [this](https://github.com/Financial-Times/x-dash/blob/master/private/blueprints/component/stories/index.js#L7-L11) then the document should be setup as expected 👌